docs/integrations/credentials/cal.md

```diff
@@ -0,0 +1,21 @@
+# Cal
+
+You can use these credentials to authenticate the following nodes with Cal.
+
+- [Cal Trigger](/integrations/trigger-nodes/n8n-nodes-base.calTrigger/)
+
+## Prerequisites
+
+Create a [Cal](https://www.cal.com/) pro account.
+
+## Using API keys
+
+Your API requests are authenticated using API keys. Any request that doesn't include an API key will return an error.
+
+1. Log into your Cal account
+2. Click on Settings
+3. There you will find the API keys section which allows you to generate a key for use. More information on this can be found on the [Cal Authentication](https://developer.cal.com/api/authentication) page
```

Comment:
```suggestion
3. There you will find the API keys section which allows you to generate a key for use. More information on this can be found on the [Cal Authentication](https://developer.cal.com/api/authentication){:target=_blank} page.
```